app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+  def new
+    @likes = Like.new
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find(params[:post_id])
+    @like = @post.likes.build(user: current_user)
+
+    if @like.save
+      redirect_to user_post_path(@user, @post), notice: 'Like was sucessfully created.'
+    else
+      render :new
+    end
```

Comment:
- [ ] The like should created using the `current_user`, not the user from the URL